src/app.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = () => {
```

Comment:
The 'copyFile' function should take 'file' and 'copy' as parameters instead of accessing 'process.argv' directly.